packages/create-pwa/lib/index.js

```diff
@@ -1,17 +1,38 @@
 const { basename, resolve } = require('path');
 const os = require('os');
+const fetch = require('node-fetch');
 const changeCase = require('change-case');
 const inquirer = require('inquirer');
 const execa = require('execa');
 const chalk = require('chalk');
 const gitUserInfo = require('git-user-info');
 const isInvalidPath = require('is-invalid-path');
 const isValidNpmName = require('is-valid-npm-name');
+const { uniqBy } = require('lodash');
+
 const pkg = require('../package.json');
 const {
-    sampleBackends
+    sampleBackends: defaultSampleBackends
 } = require('@magento/pwa-buildpack/lib/cli/create-project');
 
+const removeDuplicateBackends = backendEnvironments =>
+    uniqBy(backendEnvironments, 'url');
+
+const fetchSampleBackends = async () => {
+    try {
+        const res = await fetch(
+            'https://fvp0esmt8f.execute-api.us-east-1.amazonaws.com/default/getSampleBackends'
+        );
+        const { sampleBackends } = await res.json();
+
+        return sampleBackends.environments;
+    } catch (err) {
+        console.error(err);
```

Comment:
I think this failure needs to be silent outside of debug environments. This would be surfaced to someone trying to scaffold an app for the first time, and they aren't able to resolve this failure.

Reply:
I got a 👍  on this suggestion but it hasn't been resolved. I still see errors output to the user if you could please resolve.